Purchases/Networking/HTTPClient.swift

```diff
@@ -21,6 +21,7 @@ class HTTPClient {
     private var currentSerialRequest: HTTPRequest?
     private var eTagManager: ETagManager
     private let operationDispatcher: OperationDispatcher
```

Comment:
is `operationDispatcher` still used in this class? if not, can we remove it?

Reply:
Oh hey, look at that. Yeah!

Reply:
✅